src/MeetingsSDKAdapter.test.js

```diff
@@ -62,22 +64,19 @@ describe('Meetings SDK Adapter', () => {
   });
 
   describe('getLocalMedia()', () => {
-    beforeEach(() => {
-      global.MediaStream = jest.fn((instance) => instance);
-    });
-
     test('returns local media in a proper shape', (done) => {
       meetingSDKAdapter.getLocalMedia(meetingID).pipe(last()).subscribe((dataDisplay) => {
-        expect(dataDisplay).toMatchObject({
-          localAudio: {
-            stream: mockSDKMediaStreams.localAudio,
-            permission: 'ALLOWED',
-          },
-          localVideo: {
-            stream: mockSDKMediaStreams.localVideo,
-            permission: 'ALLOWED',
-          },
-        });
+        expect(dataDisplay.localAudio.stream.getTracks()).toMatchObject(
```

Comment:
you can define a custom matcher for media streams:
```js
expect(displayData.localAudio.stream).toMatchMediaStream(mockSDKMediaStreams.localAudio);
```
see https://jestjs.io/docs/expect#custom-matchers-api